packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -547,3 +547,814 @@ declare module 'lightning/uiRecordApi' {
+        time?: TimeRepresentation;
+    }
+
+    /**
+     * Time at which something should happen
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#TimeRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface TimeRepresentation {
+        /** Hour at which this schedule is run (0-23). */
+        hour: number;
+        /** Minute at which this schedule is run (0-59). */
+        minute: number;
+        /** Time zone of the hour at which the schedule is run. */
+        timeZone: TimeZoneRepresentation;
+    }
+
+    /**
+     * Information about a time zone.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#TimeZoneRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface TimeZoneRepresentation {
+        /** The signed offset, in hours, from GMT. */
+        gmtOffset: number;
+        /** The display name of this time zone. */
+        name: string;
+        /** The zone ID of this time zone. */
+        zoneId: string;
+    }
+
+    /**
+     * Information about a user.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#WaveUserRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface WaveUserRepresentation {
+        /** The 18 character user ID. */
+        id: string;
+        /** The name of the user. */
+        name?: string;
+        /** The Chatter profile photo of the user. */
+        profilePhotoUrl?: string | null;
+    }
+
+    /**
+     * Weekly schedule on which to execute some type of job.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#WeeklyScheduleInputRepresentation
+     *
+     * Keys:
+     *    (none)
+     */
+    export interface WeeklyScheduleInputRepresentation extends ScheduleInputRepresentation {
+        frequency: 'weekly' | 'Weekly';
+        /** Days of the week on which the schedule will run. */
+        daysOfWeek: Array<string>;
+    }
+
+    /**
+     * Weekly schedule on which to execute some type of job.
+     *
+     * https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_appendix.htm#WeeklyScheduleRepresentation
+     *
+     * Keys:
+     *    id (string): assetId
+     */
+    export interface WeeklyScheduleRepresentation extends ScheduleRepresentation {
+        frequency: 'weekly' | 'Weekly';
+        /** Days of the week on which the schedule will run. */
+        daysOfWeek: Array<string>;
+    }
+
+    /**
+     * Creates a Tableau CRM dataflow job, which is the equivalent of clicking Run Now for a data prep recipe, a data sync,
+     * or a dataflow in the Tableau CRM Data Manager UI.
+     *
```

Comment:
The Developer Guide docs for these are currently in progress. Once their published and public, I'll update these methods here with links to them.